packages/server-web/src/client/component/TodoItem.vue

```diff
@@ -186,14 +174,11 @@ export default {
 }
 .dropzone-sub {
   position: absolute;
-  top: -10px;
+  top: 10px;
```

Comment:
I think this could be `top: 0;` here. I would not want to "loose" what I've dragged along for 3 pages just because I've missed the 10px between two todos, don't you think?

Reply:
I tried first with 0px but had the impression, that the dropzone was still above the todo. will try again...

Reply:
you're right, 0 works perfectly fine. will change it to `0`